TODO.md

```diff
@@ -0,0 +1,8 @@
+- [ ] Make cursor appearance indicate condition such as unable to place a gate
```

Comment:
Can you put these into GitHub issues and remove this file?

Reply:
Removed file, will open as issues soon.